frontend/src/components/MapView/DateSelector/TimelineItems/index.tsx

```diff
@@ -114,28 +131,85 @@ const TimelineItems = memo(
       [classes.dateItemLabel, classes.dayItem, locale],
     );
 
+    const concatenatedLayers = useMemo(() => {
+      const timelineStartDate: string = new Date(
+        dateRange[0].value,
+      ).toDateString();
+
+      const hello = [intersectionDateItems, ...selectedLayerDateItems].map(
```

Comment:
Why the name is `hello` also this would be better if it was a `useMemo` outside of `concatenatedLayers`

Reply:
Done